### PR TITLE
GAWB-4079: Add get_cost action on workflow-collection resource

### PR DESCRIPTION
**Pull request by ansingh7115**
Ticket: https://broadinstitute.atlassian.net/browse/GAWB-4079

---

**PR checklist**

- [ ] I've followed [the instructions](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

### CODE REVIEW

File: src/main/resources/reference.conf [hunk 1]
```diff
@@ -187,13 +190,13 @@ resourceTypes = {
     ownerRoleName = "owner"
     roles = {
       owner = {
-        roleActions = ["delete", "add", "view", "abort", "update", "alter_policies", "read_policies"]
+        roleActions = ["delete", "add", "view", "abort", "get_cost", "update", "alter_policies", "read_policies"]
       }
       reader = {
         roleActions = ["view"]
```
Comment (dvoet) [inline src/main/resources/reference.conf]: should reader should have get_cost too?

File: src/main/resources/reference.conf [hunk 2]
```diff
@@ -174,6 +174,9 @@ resourceTypes = {
       abort = {
         description = "abort any workflow in this workflow-collection"
       }
+      get_cost = {
+        description = "get the cost of a any workflow in this workflow-collection"
```
Comment (dvoet) [inline src/main/resources/reference.conf]: drop `a`

File: src/main/resources/reference.conf (hunk 1 above)
Comment (ansingh7115) [reply src/main/resources/reference.conf]: I'm not sure yet - I was thinking no and then we add it if people want it. My logic was - if someone doesn't own the workspace or have the ability to run anything in it - why do they care about the costs?

Comment (ansingh7115) [reply src/main/resources/reference.conf]: is it complicated to remove an action from a role? If not, I don't mind readers having get_cost

**Review by dvoet — APPROVED** [review]

**Pull request merged**